merlin/config/results_backend.py

```diff
@@ -60,10 +60,7 @@
 
 
 MYSQL_CONNECTION_STRING = (
-    "db+mysql+mysqldb://{user}:{password}@{server}/mlsi"
-    "?ssl_ca={ssl_ca}"
-    "&ssl_cert={ssl_cert}"
-    "&ssl_key={ssl_key}"
+    "db+mysql+mysqldb://{user}:{password}@{server}/mlsi" "?ssl_ca={ssl_ca}" "&ssl_cert={ssl_cert}" "&ssl_key={ssl_key}"
```

Comment:
So, yes, it doesn't need them, the string will be read as one token rather than each separate token, but, I see why it had those breaks, it was for readability - each line is basically a separate var: the login credentials, who the ssl certificate authority is, the actual ssl certificate, and the ssl key. I'll think about whether that's worth preserving.
```suggestion
    "db+mysql+mysqldb://{user}:{password}@{server}/mlsi?ssl_ca={ssl_ca}&ssl_cert={ssl_cert}&ssl_key={ssl_key}"
```

Reply:
I think it makes sense to keep that formatting, because otherwise it's a big, long, mooshed together string, so I added markup for Black to skip this chunk.